packages/config/config/devices/0x010f/fgk101.json

```diff
@@ -103,17 +112,26 @@
 		{
 			"#": "2",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "Door/Window or alarm status",
-			"description": "Parameter defines state of the sensor when the magnet ...",
+			"label": "Door State when Magnet is Close",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Closed",
+					"value": 0
+				},
+				{
+					"label": "Open",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "3",
-			"$if": "firmwareVersion <= 2.4 && firmwareVersion <= 3.1",
-			"label": "Type of input no. 1",
+			"$if": "firmwareVersion <= 3.1",
```

Comment:
Please double-check the firmware version. `<= 2.4 && <= 3.1` would logically simplify to `<= 2.4`